code/modules/reagents/chemistry/recipes/cat2_medicines.dm

```diff
@@ -61,5 +61,5 @@
 	required_reagents = list(/datum/reagent/sulfur = 1, /datum/reagent/fluorine = 1, /datum/reagent/toxin = 1, /datum/reagent/nitrous_oxide = 2)
 
 /datum/chemical_reaction/penthrite
-	results = list(/datum/reagent/medicine/C2/penthrite = 4)
-	required_reagents = list(/datum/reagent/pentaerythritol = 4, /datum/reagent/acetone = 1,  /datum/reagent/toxin/acid/nitracid = 1)
+	results = list(/datum/reagent/medicine/C2/penthrite = 3)
+	required_reagents = list(/datum/reagent/pentaerythritol = 1, /datum/reagent/acetone = 1,  /datum/reagent/toxin/acid/nitracid = 1 , /datum/reagent/ammonia = 1 , /datum/reagent/medicine/atropine = 1)
```

Comment:
you sure that the penthrite made from this won't react with the atropine that's being used to make it?

Reply:
oh wait fuck you are right